Tests/GPU/CuptiTest/Exec/CUDA/myfunc.cpp

```diff
@@ -0,0 +1,65 @@
+#include "myfunc.H"
+#include "mykernel.H"
+
+#ifdef AMREX_USE_CUPTI
+#include <AMReX_ActivityTraceAsync.H>
+#endif // AMREX_USE_CUPTI
+
```

Comment:
Depending on how we want to deal with TINY_PROFILE + CUPTI options, we may want to bury this ifdef logic in the macros themselves.

Reply:
The import is now moved to the place where CUPTI macros are defined.  If using the CuptiTrace class as a standalone, the header still needs to be imported.